pkg/volume/awsebs/aws_ebs.go

```diff
@@ -428,7 +428,7 @@ func (b *awsElasticBlockStoreMounter) SetUpAt(dir string, mounterArgs volume.Mou
 	}
 
 	if !b.readOnly {
-		volume.SetVolumeOwnership(b, mounterArgs.FsGroup, mounterArgs.FSGroupChangePolicy)
+		volume.SetVolumeOwnership(b, mounterArgs.FsGroup, mounterArgs.FSGroupChangePolicy, util.OperationCompleteHook(b.plugin.GetPluginName(), "volume_fsgroup_recursive_apply"))
```

Comment:
Please use a const for "volume_fsgroup_recursive_apply". Or add `util.FSGroupCompleteHook()` into volume_linux.go 
 that would return `OperationCompleteHook` with the right op. name.

Reply:
done.